tap_zendesk/streams.py

```diff
@@ -139,88 +141,110 @@ def sync(self, state):
 class Tickets(Stream):
     name = "tickets"
     replication_method = "INCREMENTAL"
-    replication_key = "updated_at"
+    replication_key = "generated_timestamp"
 
     last_record_emit = {}
-    def _buffer_record(self, buf, record):
+    buf = {}
+    buf_time = 60
```

Comment:
I was thinking about this. I like it, keep the timer to align the same with the parent stream's metric emitting frequency.